src/image.rs

```diff
@@ -190,4 +218,27 @@ mod test {
         let actual: TestImage = serde_yaml::from_str(yaml).unwrap();
         assert!(actual == expected);
     }
+
+    #[test]
+    fn test_image_spec_by_exec_spec() {
+        let yaml = r#"
+image:
```

Comment:
```
image:
  build:
    custom:
       ... your new exec stuff...
```
??

Reply:
The current buildspec has mandatory dockerfile / context fields; I didn't want to really interfere with that.